### DISCUSSION TITLE
Prism Template - Master Page

### DISCUSSION THREAD

**Discussion by nick5454** [PrismLibrary/Prism#2684] [General] [answered]
Hi,

I became a sponsor for @dansiegel, so I can access more features of Prism for my projects. When installing the Prism Template Pack. I only see the Blank Xamarin forms and not Master Page ( Flyout ). Is there a template for that in VS2022? I don't see it and maybe i'm missing something? 

After reading, I see it's not there. Is there any plans to add this to Xamarin Forms template?

**Comment by brianlagunas** [comment, marked as answer]
There is no such template. What led you to believe that Prism template existed?

**Reply by dansiegel** [reply]
@nick5454 thank you for stepping up as a Sponsor! You should have access to [Sponsor Connect](https://sponsorconnect.dev), and you can create NuGet key to connect to the private NuGet feed which has some Sponsor Only packages such as the Prism Magician, and all CI packages so that you have instant access to bug fixes or features as they are merged. There are however no special things unlocked in the Template Pack.

**Reply by nick5454** [reply]
@brianlagunas I could have sworn at one time ( maybe 2017 VS ) there was a master flyout template. Maybe thinking of something else or wishful thinking
